core/src/packets/ip/v6/srh.rs

```diff
@@ -94,7 +94,7 @@ use std::ptr::NonNull;
 ///                        SR Policy, the second element contains the
 ///                        penultimate segment of the SR Policy and so on.
 ///
-/// Type Length Value (TLV) are described in Section 2.1.
+/// - *Type Length Value*: this is not currently supported.
```

Comment:
Still would be good to say what TLV is (from the spec)? I guess it can always be updated in docs, but could be more complete.

Reply:
added a couple sentences from the spec. we can fill in the details when it's implemented.